src/guc.c

```diff
@@ -868,6 +880,19 @@ _guc_init(void)
 							/* assign_hook= */ NULL,
 							/* show_hook= */ NULL);
 
+	DefineCustomEnumVariable(/* name= */ MAKE_EXTOPTION("debug_require_vector_agg"),
+							 /* short_desc= */
+							 "ensure that vectorized aggregation is used or not",
+							 /* long_desc= */ "this is for debugging purposes",
+							 /* valueAddr= */ (int *) &ts_guc_debug_require_vector_agg,
+							 /* bootValue= */ RVQ_Allow,
+							 /* options = */ require_vector_agg_options,
```

Comment:
won't this break on release builds?

Reply:
Yeah I made some tests debug-only. We have a similar thing with `debug_require_vector_qual` and `debug_require_batch_sorted_merge`.

Reply:
Is the intention make it available for debugging production workloads as well? If the answer is yes maybe would be good to include the `config_enum_entry` for those GUC into release builds. If not then add everything inside the TS_DEBUG stuff. WDYT?

Reply:
No, this is only for tests, so that we can check that the particular plan is used, w/o running explain for each query. For manual debugging it's OK run explain.

`require_vector_agg_options` is already under `TS_DEBUG`, did I miss something else?

Reply:
Then you should to add everything under the `TS_DEBUG` including the GUC itself since it will not be available for the end user.